reference/kubectl-plugin.md

```diff
@@ -54,7 +53,7 @@ SUBCOMMANDS:
     drain       'Drain' resources
     dump        `Dump` resources
     get         'Get' resources
-    help        Print this message or the help of the given subcommand(s)
+    help         Print this message or the help of the given subcommand(s)
```

Comment:
Is the 'P' aligned with the ' above?

Reply:
Aligned it manually. However, this output is taken directly from the console hence the actual output would look like the above.

Reply:
Ok